src/lib/renderExamplePageTest.helper.js

```diff
@@ -127,7 +128,23 @@ const testSnapshots = async ({ page, testName, snapshotNames = null }) => {
 
   await asyncForEach(widgets, async (widget, index) => {
     let image = await widget.screenshot({})
-    expect(image).toMatchImageSnapshot({ customSnapshotIdentifier: getSnapshotName(index) })
+    const snapshotName = getSnapshotName(index)
+    try {
+      expect(image).toMatchImageSnapshot({ customSnapshotIdentifier: snapshotName })
+    } catch (e) {
+      // Can't find group name so just put all new snapshots in same folder
```

Comment:
Could the group name be extracted from `snapshotName`? Seems slightly inconvenient to have to find the right group when updating the snapshots.

Reply:
Unfortunately it can't be. I can see where snapshotDirectory is defined, but to get to it I would been to change a lot of the function signatures of exported functions. I think most of the time its not that annoying because if you only have a few diffs its actually nicer that they are in the same directory. But maybe a bit annoying if there are a lot of snapshots with some ambiguous names